### ISSUE TITLE
One line

### ISSUE THREAD

**Issue by Wicowyn** [Manabu-GT/ExpandableTextView#12]
When the TextView have just one line to show, the ExpandableTextView let the possibility to be collapsed. And when click the line is cut in half without possibility to expand.

**Comment by Manabu-GT** [comment]
@Wicowyn Thank you for reporting the issue. Since I could not reproduce the issue with one line of english text, can you please share the text and the layout file you used so that I can quickly reproduce the issue and fix it?

**Comment by Wicowyn** [comment]
Yes I try to reproduce it on a fresh project but it's a bit diffucult

**Comment by squeeish** [comment]
I've encountered this as well, with English text. I'll post up my code and a recording to demonstrate later today.
